docs/README.md

```diff
@@ -23,6 +25,46 @@ tbd
 1. Click **Operators** -> **Installed Operators** to verify that the sail-operator 
 is installed. `Succeeded` should appear in the **Status** column.
 
+#### Installing using the CLI
+
+*Prerequisites*
+
+* You must have `admin` privileges.
+
+*Steps*
+
+1. Create the `openshift-operators` namespace (if it does not already exist).
+
+    ```bash
+    $ kubectl get ns openshift-operators --ignore-not-found || kubectl create namespace openshift-operators
```

Comment:
Doesn't the `--ignore-not-found` option make the entire first command useless?

Reply:
Hmm, not just useless, but it actually prevents the `kubectl create` command from ever being invoked, since it always succeeds.

Reply:
You're right, going to change this

Reply:
I think you can just skip the first command. This isn't a shell script, so the user will usually know whether the namespace exists or not (and will thus know whether to run the `kubectl create` command or not). And even if they run it when they don't need to and get an error, that's not that bad. I think it's more important for the instructions to be as simple and short as possible (so, a simple `kubectl create` is better then `kubectl get || kubectl create`).

Reply:
sorry I overlooked this. will this ever be the case though? In OCP that namespace should always exist